src/Symfony/Component/EventDispatcher/EventDispatcherInterface.php

```diff
@@ -22,6 +22,11 @@
  */
 interface EventDispatcherInterface extends ContractsEventDispatcherInterface
 {
+    /**
+     * {@inheritdoc}
+     */
+    public function dispatch($event, string $eventName = null);
```

Comment:
already in the parent interface, thus not needed

Reply:
Yes, but in the parent interface, the second argument is commented out. As far as I understand [this note](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Contracts/EventDispatcher/EventDispatcherInterface.php#L25-L27), event dispatcher implementations in 5.0 must have this signature, so I'm making sure that they do.